### PR TITLE
Create Org Settings Page

### PR DESCRIPTION
**Pull request by crabsinger**
Create a new Settings page for Organizations:
- supports changing org name, telegram_id, and logo.
- add link to settings page for Orgs page
Screen Shot 2022-10-07 at 5 40 14 PM

Screen Shot 2022-10-07 at 5 39 35 PM

## Testing:

- File uplaod on all pages including profiles, as changed some of that logic
- Test form behavior
- Check migrations

TODOs:
- [ ] verify the contraints will be valid against existing prod data

Changes:
- Remove unneccessary Box in SavingIndicator
- Add migrations with db constraints
- update org permissions
- create org settings page
- add routes
- fix express file upload size limit
- update types
- Remove unneeded useEffect
- Add error handling to uploadOrgLogo

### CODE REVIEW

File: api/hasura/actions/_handlers/uploadOrgLogo.ts [hunk 1]
```diff
@@ -26,11 +26,15 @@ const handler = async function (req: VercelRequest, res: VercelResponse) {
...
-  );
-  return res.status(200).json(updatedProfile);
+  try {
+    const updatedProfile = await updater.uploadImage(
+      input.image_data_base64,
+      previousLogo
+    );
+    return res.status(200).json(updatedProfile);
+  } catch (e: any) {
+    return res.status(400).json('error: ' + e?.message || 'unknown');
```
Comment (crabsinger) [inline api/hasura/actions/_handlers/uploadOrgLogo.ts:36]: Is this going to leak sensitive data potentially? Maybe we shouldn't have any error passing through

File: hasura/metadata/databases/default/tables/public_organizations.yaml [hunk 2]
```diff
@@ -43,3 +44,26 @@ select_permissions:
...
+                  - profile:
+                      id:
+                        _eq: X-Hasura-User-Id
+                  - deleted_at:
+                      _is_null: true
+                  - role:
+                      _eq: 1
+      check:
+        name:
+          _regex: ^.{3,70}$
```
Comment (crabsinger) [inline hasura/metadata/databases/default/tables/public_organizations.yaml:69]: Could be used in place of the DB constraints...

File: src/hooks/useImageUploader.tsx [hunk 3]
```diff
@@ -1,12 +1,8 @@
-import { useState, useEffect } from 'react';
+import { useState } from 'react';
 
 export const useImageUploader = (original: string) => {
   const [file, setFile] = useState<File | undefined>();
 
-  useEffect(() => {
```
Comment (crabsinger) [inline src/hooks/useImageUploader.tsx:6]: Tested this a bit and saw no need for this

**Comment by wingfeatherwave** [comment]
Something seems to be missing.  I'm not able to click on an Org name and visit an org page.  clicking to `/organizations/16` for example redirects me back to `/circles` with an access denied message in the browser console, though I am an admin of that circle.

**Review by wingfeatherwave — APPROVED** [review]

File: api/hasura/actions/_handlers/uploadOrgLogo.ts (hunk 1 above)
Comment (topocount) [inline api/hasura/actions/_handlers/uploadOrgLogo.ts:36]: @crabsinger you'll want to familiarize yourself with how we catch and handle errors here: https://github.com/coordinape/coordinape/blob/contrib-integation-daterange/api-lib/HttpError.ts

**Pull request merged**